src/event.rs

```diff
@@ -154,4 +195,10 @@ pub enum Kind {
 }
 
 #[derive(Serialize, Deserialize, PartialEq, Debug, Clone)]
-pub struct Tag {}
+pub struct Tag([String; 3]);
+
+impl Tag {
+    pub fn new(kind: &str, content: &str, recommended_relay_url: &str) -> Self {
```

Comment:
I believe the recommended_relay_url is optional (https://github.com/fiatjaf/nostr/blob/master/nips/01.md#other-notes). if none it can be an empty string so can keep this same `Tag` struct I think